src/SME.css

```diff
@@ -0,0 +1 @@
+.sme-col{height:30px}.sme-item{height:30px}html:not([dir=rtl]) nav ol.breadcrumb{margin-left:0 !important}/*# sourceMappingURL=SME.css.map */
```

Comment:
Are these files needed? If not, kindly remove all unnecessary files included in this PR. Only add the files that you need.